.github/workflows/build-integration.yml

```diff
@@ -0,0 +1,88 @@
+name: gocardless/stolon-pgbouncer/build-integration
+on:
+  push:
+env:
+  DOCKER_PASS: xxxx9e14
+  DOCKER_USER: xxxxdmin
+  GITHUB_TOKEN: xxxx6d08
```

Comment:
Not sure what this was set to before. We might be able to get away with the usual `GITHUB_TOKEN` injected by GHA.

Reply:
Hey Sam 👋 

A note of warning: this is a public repo that anyone can open PRs to, so you’ll want to double check you don’t respond to those changes by running builds on any private GitHub agents you may have had around.

Especially if you’re relying on ambient creds on those machines, which might mean people outside of GC could exfiltrate GitHub resources under the GC org by misusing the creds.

I’ve not checked this super closely, but wanted to warn you in case you hadn’t considered it. It may make more sense to leave this and other open source repos on Circle, to avoid exposing the private infra.

Reply:
Hey Lawrence,

Thanks for the heads up! 
We don't have self-hosted runners set up just yet, but when we do we'll restrict them to run solely on private repositories, and leave public repos to use Github's runners.

Reply:
Great, wanted to nudge you in case it had gone under the radar, but sounds like you’re on it :)